recipes/pytables/bld.bat

```diff
@@ -0,0 +1,26 @@
+REM Would be nice to include blosc support, but building this with external blosc is currently broken.
+REM    See https://github.com/PyTables/PyTables/issues/451
+
+
+REM rd /s /q c-blosc
+REM git clone https://github.com/Blosc/c-blosc
+REM cd c-blosc
+REM git checkout v1.7.0
+REM git clone https://github.com/Blosc/hdf5-blosc
```

Comment:
See the comment a few lines up.  Pytables can't currently utilize these.  Yes, it's worth packaging them, and I'll do so if someone beats me to it. However, we can't use them here.